runner/AndroidJunitRunnerSample/app/src/androidTest/java/com/example/android/testing/androidjunitrunnersample/CalculatorAddParameterizedTest.java

```diff
@@ -88,7 +85,7 @@ public void setUp() {
     @Test
     public void testAdd_TwoNumbers() {
         double resultAdd = mCalculator.add(mOperandOne, mOperandTwo);
-        assertThat(resultAdd, is(equalTo(mExpectedResult)));
+        assertEquals(resultAdd, mExpectedResult, 0);
```

Comment:
What about using Google Truth to replace API to `assertThat(resultAdd).isEquals(mExpectedResult)`? Maybe it could provide more clear error information than normal JUnit assertion API.

Reply:
As the code base is stable now, I think there is no need to introduce another library here to get `more clear error information` 😂.

Reply:
It's just a nit hint. IMO, this PR improves current code base's quality, and Google Truth is a better choice for current used APIs, recommended by Google. It's not a problem.

Reply:
> I can take the AI to update the samples to truth. The samples should be using truth and its wrapper androidx.test.ext.truth where appropriate

Somehow I cannot checkout my fix/deprecated-methods branch now.
 I'll give up this PR, and start another one.